### DISCUSSION TITLE
解决 Java 打印日志吞异常堆栈的问题 — 码志

### DISCUSSION THREAD

**Discussion by giscus** [mzlogin/blog-comments#267] [Announcements]
# 解决 Java 打印日志吞异常堆栈的问题 — 码志

前几天有同学找我查一个空指针问题，Java 打印日志时，异常堆栈信息被吞了，导致定位不到出问题的地方。

https://mazhuang.org/2023/06/26/java-swallow-stacktrace/

**Comment by Beeta** [comment]
不错
